### ISSUE TITLE
WordSets

### ISSUE THREAD

**Issue by blegat** [blegat/openvoc#18]
I have a problem with the current implementation of WordSet.

Why is the WordSet related to the pair (Word, Meaning) and not directly to the corresponding Link between this word and this meaning ?

**Comment by pverbist95** [comment]
Yes, you are right: the WordSet should be linked directly to the Link.

**Comment by blegat** [comment]
I will add the method `word` and `meaning` in the Model so that everything keeps working :)
